pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -325,7 +325,8 @@ func (rq *ResourceQuotaController) syncResourceQuota(v1ResourceQuota *v1.Resourc
 			Namespace:       resourceQuota.Namespace,
 			ResourceVersion: resourceQuota.ResourceVersion,
 			Labels:          resourceQuota.Labels,
-			Annotations:     resourceQuota.Annotations},
+			Annotations:     resourceQuota.Annotations,
```

Comment:
to avoid this problem with any other field introduced in the future, can we do the following:
```
usageStatus := api.ResourceQuotaStatus{
Hard: hardLimits,
Used: used
}

dirty = dirty || !quota.Equals(usageStatus.Used, resourceQuota.Status.Used)
if dirty {
usage := DeepCopy(resourceQuota)
v1Usage := &v1.ResourceQuota{}
if err := v1.Convert_api_ResourceQuota_To_v1_ResourceQuota(&usage, v1Usage, nil); err != nil {
	return err
}
_, err = rq.kubeClient.Core().ResourceQuotas(usage.Namespace).UpdateStatus(v1Usage)
}
```

above is pseudo-code, but hopefully the idea is clear?